include/RAJA/pattern/atomic.hpp

```diff
@@ -329,7 +329,7 @@ class AtomicRef
 
   RAJA_INLINE
   RAJA_HOST_DEVICE
-  value_type volatile * getPointer() const { return m_value_ptr; }
+  value_type * getPointer() const { return m_value_ptr; }
 
   RAJA_INLINE
```

Comment:
These store and load functions should be properly atomic.

Reply:
Good catch! Fixed.